module-apps/application-calllog/ApplicationCallLog.cpp

```diff
@@ -133,7 +133,9 @@ namespace app
         DBServiceAPI::GetQuery(this,
                                db::Interface::Name::Notifications,
                                std::make_unique<db::query::notifications::Clear>(NotificationsRecord::Key::Calls));
-        return true;
+        const auto [succeed, _] = DBServiceAPI::GetQuery(
+            this, db::Interface::Name::Calllog, std::make_unique<db::query::calllog::SetAllRead>());
+        return succeed;
```

Comment:
so indeed it was needed?

Reply:
yep, I missed it. It was needed for dot in menu window, which is marker of unread calls in calllog